### PR TITLE
Article about using DNSimple as slave

### LINKED ISSUE #211: Article about using DNSimple as secondary via the API
It is possible to use DNSimple as a secondary today by enabling secondary DNS and then using the API to send in changes to a zone. This article should explain how to do this.

### PR DESCRIPTION
**Pull request by jacegu**
Closes #211 

We have also promoted the Secondary articles to their own category to make them easier to spot (and because the DNS category has too many things in it).

### CODE REVIEW

**Comment by sbastn** [comment]
Ready for your 👀

File: content/articles/secondary-dnsimple-slave.markdown [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+- [Using our API](https://developer.dnsimple.com/v2).
+
+Once you are able to keep both zones in sync you will need to add the NS records corresponding to your primary DNS provider. You won't be able to do this using our [record editor](/articles/record-editor/) directly.
+
+
+## Configuring DNSimple as a slave
+
+These are the steps required in order to setup DNSimple as a slave for your domain.
+
+- If you don't have the domain in your DNSimple account, [add it with "use DNSimple services"](/articles/adding-domain/).
```
Comment (jcaudle) [inline content/articles/secondary-dnsimple-slave.markdown]: Should we put these bullet points in a `steps` div?

**Comment by jacegu** [comment]
One thing we may want to change as part of this PR is what we say about using dnsimple as a Slave in the other articles. Mostly this: https://support.dnsimple.com/articles/secondary-dns/#frequently-asked-questions

File: content/articles/secondary-dnsimple-slave.markdown (hunk 1 above)
Comment (sbastn) [inline content/articles/secondary-dnsimple-slave.markdown]: @jcaudle This article is temporary until we deploy the new flow that @jacegu and I are currently working on. We expect to have it ready this week. So the steps might not be needed anymore.

**Comment by sbastn** [comment]
@weppos @aeden Have a look at the last commit and let us know if this expresses correctly our concept of Secondary DNS.

File: content/articles/secondary-dnsimple-slave.markdown [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+excerpt: How to use DNSimple along with other DNS providers to implement zone redundancy.
+categories:
+- Secondary DNS
+---
+
+# DNSimple as a Secondary DNS provider
+
+While DNSimple cannot be configured as Secondary slave for your domain, this article explains how you can have zone redundancy with other DNS providers.
+
+We currently do not implement zone transfers (AXFR) from another primary DNS provider. That means you will be responsible of keeping the zones in in sync between your primary and DNSimple. You can accomplish this in several ways:
```
Comment (aeden) [inline content/articles/secondary-dnsimple-slave.markdown]: Combine the previous paragraph with this one using the following:

> While DNSimple cannot synchronize zone changes from other name servers using AXFR, you can combine our secondary DNS feature with our API or UI to have zone redundancy with other DNS providers. You will be responsible for keeping the zones in sync between other DNS providers and DNSimple. You can accomplish this in several ways:

File: content/articles/secondary-dnsimple-slave.markdown [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+- [Adding the records](/articles/record-editor/) manually through our UI.
+- [Importing a zone file](/articles/zone-files/#importing-records-from-a-zone-file) from your primary provider.
+- [Using our API](https://developer.dnsimple.com/v2).
+
+Once you are able to keep both zones in sync you will need to add the NS records corresponding to your primary DNS provider. You won't be able to do this using our [record editor](/articles/record-editor/) directly.
```
Comment (aeden) [inline content/articles/secondary-dnsimple-slave.markdown]: > To do this you will need to use our Secondary DNS configuration feature.

Comment (sbastn) [inline content/articles/secondary-dnsimple-slave.markdown]: ✔️

**Pull request merged**